src/js/trove/multiple-regression.js

```diff
@@ -0,0 +1,182 @@
+({
+  requires: [],
+  nativeRequires: ["pyret-base/js/namespace"],
+  provides: {
+    values: {
+      "multiple-regression": "tany"
+    }
+  },
+  theModule: function(runtime, namespace, uri) {
+
+    function makeColumnMatrix(y_s) {
+      let n = y_s.length;
+      let M = new Array(n);
+      for (let i = 0; i < n; i++) {
+        M[i][0] = y_s[i];
+      }
+      return M;
+    }
+
+    function matrixTranspose(m) {
+      let numrows = m.length;
+      let numcols = m[0].length;
+      let mT = new Array(numcols);
+      for (let i = 0; i < numcols; i++) {
```

Comment:
I can tell this loop is correct, but a strong naming pet-peeve: use `r` and `c` (for row and col) instead of `i` and `j`, since they're much easier to tell apart while reading, and it helps keep track that the matrix is in row-major order.

Reply:
I've pushed some better naming. In this case (transpose), I'm not sure whether you want the `r, c` to range over the input rows/cols or the output ones (since they are exact flips of each other). I just picked one without much conviction.